scripts/us_bea/states_gdp/import_data.py

```diff
@@ -0,0 +1,134 @@
+ # Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Pulls data from the US Bureau of Economic Analysis (BEA) on quarterly GDP
+per US state. Saves output as a CSV file.
+
+    Typical usage:
+
+    python3 import_data.py
+"""
+from urllib.request import urlopen
+from absl import app
+import pandas as pd
+import io
+import zipfile
+import csv
+import re
+
+# Suppress annoying pandas DF copy warnings.
+pd.options.mode.chained_assignment = None # default='warn'
+
+US_STATES = ['Alabama', 'Alaska', 'Arizona', 'Arkansas',
+             'California', 'Colorado', 'Connecticut', 'Delaware',
+             'District of Columbia', 'Florida', 'Georgia', 'Hawaii', 'Idaho',
+             'Illinois', 'Indiana', 'Iowa', 'Kansas', 'Kentucky', 'Louisiana',
+             'Maine', 'Maryland', 'Massachusetts', 'Michigan', 'Minnesota',
+             'Mississippi', 'Missouri', 'Montana', 'Nebraska', 'Nevada',
+             'New Hampshire', 'New Jersey', 'New Mexico', 'New York',
+             'North Carolina', 'North Dakota', 'Ohio', 'Oklahoma', 'Oregon',
+             'Pennsylvania', 'Rhode Island', 'South Carolina', 'South Dakota',
+             'Tennessee', 'Texas', 'Utah', 'Vermont', 'Virginia', 'Washington',
+             'West Virginia', 'Wisconsin', 'Wyoming']
+
+
+class StateGDPDataLoader:
+    """Pulls GDP state data from BEA.
+
+    Attributes:
+        df: DataFrame (DF) with the cleaned data.
+    """
+    ZIP_LINK = "https://apps.bea.gov/regional/zip/SQGDP.zip"
+    FILE = "SQGDP1__ALL_AREAS_2005_2019.csv"
+
+    def __init__(self):
+        """Downloads the data, cleans it and stores it in instance DF."""
```

Comment:
```suggestion
        """Downloads the data, cleans it and stores it in instance data frame.
        
        Returns an instance only if it succeeds.  Otherwise, it dies.
        """
```

Reply:
Note this function doesn't actually return anything; it just saved the resulting df in the instance.